.pre-commit-config.yaml

```diff
@@ -1,10 +1,4 @@
 repos:
-- repo: https://github.com/asottile/add-trailing-comma
-  rev: v0.7.0
-  hooks:
-  - id: add-trailing-comma
-    # TODO: Remove this line. For now, we only format the api/ directory
-    files: ^api/
```

Comment:
... I think the above comment refers only to _this_ line, not the whole `add-trailing-comma` block.